### PR TITLE
add properties to override system views location

### PR DESCRIPTION
**Pull request by momiji**
This PR allows to change system views location, for people using other template engine for example.

Even if it was possible by extending NinjaDefault, this is much simplier and much "ninja" way of doing things, isn't it?

### CODE REVIEW

**Comment by momiji** [comment]
OK this PR is finished, can be reviewed and merged ;)

File: ninja-core/src/test/java/ninja/NinjaDefaultTest.java [hunk 1]
```diff
@@ -318,6 +318,11 @@ public void testThatGetInternalServerErrorDoesFallsBackToHtml() throws Exception
     @Test
     public void getInternalServerErrorResult() throws Exception {
         
+        when(ninjaProperties.getWithDefault(
+                Matchers.eq(NinjaConstant.LOCATION_VIEW_HTML_INTERNAL_SERVER_ERROR_KEY), 
+                Matchers.eq(NinjaConstant.LOCATION_VIEW_FTL_HTML_INTERNAL_SERVER_ERROR)))
+                .thenReturn(NinjaConstant.LOCATION_VIEW_FTL_HTML_INTERNAL_SERVER_ERROR);
```
Comment (raphaelbauer) [inline ninja-core/src/test/java/ninja/NinjaDefaultTest.java:324]: can you add a verify at the end of each test that makes sure that getWithDefault was really called?

**Comment by raphaelbauer** [comment]
Very nice. I added one comment to extend the testcase. Otherwise it is quite nice!

Please also add the change to changelog.md. Yourself to team.md. And - did you already sign the contributor's agreement? (http://www.ninjaframework.org/developer/getting_started.html)

**Comment by momiji** [comment]
Done for all PRs

**Comment by raphaelbauer** [comment]
very nice :)

Your other PRs are in the pipeline and will be reviewed step by step... :)

**Pull request merged**